judo-jsl-fullstack-karaf-project-template-application/src/main/resources/rest/pom.xml.hbs

```diff
@@ -0,0 +1,169 @@
+                        <uri>mvn:hu.blackbelt.judo.generator:judo-psm-generator-jaxrs-api:${judo-psm-generator-jaxrs-version}</uri>
+                        <uri>mvn:hu.blackbelt.judo.generator:judo-psm-generator-jaxrs-impl:${judo-psm-generator-jaxrs-version}</uri>
+                        <uri>mvn:hu.blackbelt.judo.generator:judo-psm-generator-jaxrs-osgi:${judo-psm-generator-jaxrs-version}</uri>
+                    </uris>
+                    <type>psm-jaxrs</type>
+                    <contextAccessor>hu.blackbelt.judo.psm.generator.jaxrs.api.StoredVariableHelper</contextAccessor>
+                    <scanPackages>
+                        hu.blackbelt.judo.generator.commons,
+                        hu.blackbelt.judo.psm.generator.jaxrs
+                    </scanPackages>
+                    <templateParameters>
+                        <debugPrint>true</debugPrint>
+                        <apiPrefix>${project.groupId}.{{ lowerCase model.name }}.api</apiPrefix>
+                        <implPrefix>${project.groupId}.{{ lowerCase model.name }}.impl</implPrefix>
+                        <osgiPrefix>${project.groupId}.{{ lowerCase model.name }}.osgi</osgiPrefix>
+                    </templateParameters>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>execute-psm-rest-generation</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>${basedir}/target/generated-sources</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.swagger.core.v3</groupId>
+                <artifactId>swagger-maven-plugin</artifactId>
+                <version>2.2.19</version>
+                <executions>
+                    <execution>
+                        <configuration>
+                            <outputFileName>openapi</outputFileName>
+                            <outputPath>${project.build.directory}/api-documentation-swagger</outputPath>
+                            <outputFormat>JSONANDYAML</outputFormat>
```

Comment:
Do we need both? Would it improve the generator performance if we only generated one of these?